### PR TITLE
adds harness cloud cost management integration

### PR DESCRIPTION
**Pull request by akashbdj**
### What does this PR do?

Adds harness cloud cost management integration

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: harness_cloud_cost_management/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+If you have not yet started utilizing Harness Cloud Cost Management, first [sign up](https://app.harness.io/auth/#/signup/) for a risk-free 14 day trial.
+
+For existing Harness customers all that is required is
+to complete these steps:
+
+- Install the integration
+- Go to the Harness Cloud Cost Management Dashboard and connect in any of the widgets. This will authenticate you for all the widgets
+- Log in with your Harness user
+
+## Support
```
Comment (kayayarai) [inline harness_cloud_cost_management/README.md:35]:
```suggestion
## Data Collected

### Metrics

See [metadata.csv][3] for a list of metrics provided by this check.

### Events

The Ambassador check does not include any events.

### Service Checks

The Ambassador check does not include any service checks.

## Troubleshooting

Need help? Contact [Datadog support][2].

[1]: https://app.harness.io/auth/#/signup/
[2]: https://docs.datadoghq.com/help/
```

File: harness_cloud_cost_management/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+You can reach out to us at support@harness.io
```
Comment (kayayarai) [inline harness_cloud_cost_management/README.md:37]:
```suggestion
```

File: harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json [hunk 3]
```diff
@@ -0,0 +1,193 @@
+{
+  "title": "Cloud Cost Management by Harness",
+  "description": "Harness delivers intelligent cloud cost management, providing complete cost transparency across Infrastructure, Engineering, and Finance teams with robust business intelligence, and the ability to reduce wasted cloud costs by up to 75% with Intelligent Cloud AutoStopping and ML-based recommendations.\n\n[Learn more about Cloud Cost Management ↗](https://harness.io/products/cloud-cost/)\n\n### Blog posts that might be relevant to you:\n\n### [Go to your Harness Environment ↗](https://app.harness.io)\n\n[Cloud Cost Management Concepts ↗](https://ngdocs.harness.io/category/nprch36pwj-ccm-concepts)\n\n[How-to Guides  ↗](https://ngdocs.harness.io/category/iul2qmg1yk-ccm-how-tos)\n\n[FAQs ↗](https://ngdocs.harness.io/category/yigkyv90jy-ccm-faqs)\n\n\n",
```
Comment (kayayarai) [inline harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json:3]:
```suggestion
  "description": "Harness provides cloud cost management and transparency across Infrastructure, Engineering, and Finance teams with robust business intelligence, and the ability to reduce wasted cloud costs with Intelligent Cloud AutoStopping and machine learning-based recommendations.\n\n[Learn more about Cloud Cost Management ↗](https://harness.io/products/cloud-cost/)\n\n### Blog posts that might be relevant to you:\n\n### [Go to your Harness Environment ↗](https://app.harness.io)\n\n[Cloud Cost Management Concepts ↗](https://ngdocs.harness.io/category/nprch36pwj-ccm-concepts)\n\n[How-to Guides  ↗](https://ngdocs.harness.io/category/iul2qmg1yk-ccm-how-tos)\n\n[FAQs ↗](https://ngdocs.harness.io/category/yigkyv90jy-ccm-faqs)\n\n\n",
```

File: harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+              "vertical_align": "center",
+              "horizontal_align": "center"
+            },
+            "layout": { "x": 0, "y": 0, "width": 6, "height": 4 }
+          },
+          {
+            "id": 1507941326386794,
+            "definition": {
+              "type": "note",
+              "content": "Harness delivers intelligent cloud cost management, providing complete cost transparency across Infrastructure, Engineering, and Finance teams with robust business intelligence, and the ability to reduce wasted cloud costs by up to 75% with Intelligent Cloud AutoStopping and ML-based recommendations.\n\n#### [Learn more about Cloud Cost Management ↗](https://harness.io/products/cloud-cost/)",
```
Comment (kayayarai) [inline harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json:93]:
```suggestion
              "content": "Harness provides cloud cost management and transparency across Infrastructure, Engineering, and Finance teams with robust business intelligence, and the ability to reduce wasted cloud costs with Intelligent Cloud AutoStopping and machine learning-based recommendations.\n\n#### [Learn more about Cloud Cost Management ↗](https://harness.io/products/cloud-cost/)",
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Documentation review

File: harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+      "layout": { "x": 6, "y": 0, "width": 6, "height": 7 }
+    },
+    {
+      "id": 1235094661251036,
+      "definition": {
+        "title": "",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "custom",
+        "app_id": "88f8f8cf-0a6e-11ec-bff2-da7ad0900002",
```
Comment (mgarabed) [inline harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json:132]:
```suggestion
        "app_id": "3eb2e9ef-2c9c-45b6-8f1c-8a900910f948",
```

File: harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+      "layout": { "x": 0, "y": 0, "width": 6, "height": 5 }
+    },
+    {
+      "id": 5241301662630174,
+      "definition": {
+        "title": "",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "custom",
+        "app_id": "88f8f8cf-0a6e-11ec-bff2-da7ad0900002",
```
Comment (mgarabed) [inline harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json:144]:
```suggestion
        "app_id": "3eb2e9ef-2c9c-45b6-8f1c-8a900910f948",
```

File: harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 5082626485666362,
+            "definition": {
+              "title": "",
+              "title_size": "16",
+              "title_align": "left",
+              "type": "custom",
+              "app_id": "88f8f8cf-0a6e-11ec-bff2-da7ad0900002",
```
Comment (mgarabed) [inline harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json:165]:
```suggestion
              "app_id": "3eb2e9ef-2c9c-45b6-8f1c-8a900910f948",
```

File: harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+            "layout": { "x": 0, "y": 0, "width": 5, "height": 6 }
+          },
+          {
+            "id": 6666866737269702,
+            "definition": {
+              "title": "",
+              "title_size": "16",
+              "title_align": "left",
+              "type": "custom",
+              "app_id": "88f8f8cf-0a6e-11ec-bff2-da7ad0900002",
```
Comment (mgarabed) [inline harness_cloud_cost_management/assets/dashboards/harness_cloud_cost_management_overview.json:177]:
```suggestion
              "app_id": "3eb2e9ef-2c9c-45b6-8f1c-8a900910f948",
```

File: harness_cloud_cost_management/manifest.json [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+        "media_type": "image",
+        "caption": "Cloud Cost Management Dashboard on Harness",
+        "image_url": "images/ccm_dashboard_on_harness.png"
+      }
+    ]
+  },
+  "author": {
+    "homepage": "https://www.harness.io",
+    "support_email": "akash.bhardwaj@harness.io",
+    "vendor_id": "TODO Please Fill In"
```
Comment (mgarabed) [inline harness_cloud_cost_management/manifest.json:34]:
```suggestion
```

File: harness_cloud_cost_management/manifest.json [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+      },
+      {
+        "media_type": "image",
+        "caption": "Cloud Cost Management Dashboard on Harness",
+        "image_url": "images/ccm_dashboard_on_harness.png"
+      }
+    ]
+  },
+  "author": {
+    "homepage": "https://www.harness.io",
```
Comment (mgarabed) [inline harness_cloud_cost_management/manifest.json:32]:
```suggestion
    "name": "Harness IO"
    "homepage": "https://www.harness.io",
```

Should be a name field, feel free to change to a more appropriate name.

File: harness_cloud_cost_management/manifest.json [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "logs": {}
+  },
+  "legal_terms": {}
```
Comment (mgarabed) [inline harness_cloud_cost_management/manifest.json:58]:
```suggestion
    }
```

Can delete the whole integration section.

File: harness_cloud_cost_management/metadata.csv [hunk 12]
```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```
Comment (mgarabed) [inline harness_cloud_cost_management/metadata.csv:1]: Delete this file

File: harness_cloud_cost_management/assets/service_checks.json [hunk 13]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (mgarabed) [inline harness_cloud_cost_management/assets/service_checks.json:1]: Delete this file

**Review by mgarabed — CHANGES_REQUESTED** [review]

**Comment by akashbdj** [comment]
@mgarabed Image size and aspect ratio are fixed. Please verify.

File: harness_cloud_cost_management/README.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+### Events
+
+The Ambassador check does not include any events.
+
+### Service Checks
+
+The Ambassador check does not include any service checks.
+
+## Troubleshooting
```
Comment (mgarabed) [inline harness_cloud_cost_management/README.md:36]:
```suggestion
## Support
```

File: harness_cloud_cost_management/README.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+
+2. Install the integration.
+3. Go to the Harness Cloud Cost Management Dashboard and connect in any of the widgets by logging in with your Harness user ID. This will authenticate you for all the widgets
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][3] for a list of metrics provided by this check.
+
```
Comment (mgarabed) [inline harness_cloud_cost_management/README.md:27]:
```suggestion
```

**Review by mgarabed — CHANGES_REQUESTED** [review]

File: harness_cloud_cost_management/manifest.json [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "description": "Cloud Cost Management allows Harness customers to access and view their cloud and cluster cost metrics over a specified range of dates",
+    "title": "Harness Cloud Cost Management",
```
Comment (mgarabed) [inline harness_cloud_cost_management/manifest.json:16]:
```suggestion
    "title": "Harness Cloud Cost Mgmt",
```

File: harness_cloud_cost_management/manifest.json [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "description": "Cloud Cost Management allows Harness customers to access and view their cloud and cluster cost metrics over a specified range of dates",
```
Comment (mgarabed) [inline harness_cloud_cost_management/manifest.json:15]:
```suggestion
    "description": "Cloud Cost Management allows Harness customers to access and view their cloud and cluster cost metrics over a range of dates",
```

File: harness_cloud_cost_management/README.md [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+## Setup
+
+1. If you have not yet started using Harness Cloud Cost Management, [sign up][1] for a 14 day trial.
+
+2. Install the integration.
+3. Go to the Harness Cloud Cost Management Dashboard and connect in any of the widgets by logging in with your Harness user ID. This will authenticate you for all the widgets
+
+### Events
+
+The Ambassador check does not include any events.
```
Comment (sarah-witt) [inline harness_cloud_cost_management/README.md:24]:
```suggestion
The Harness Cloud Cost Management check does not include any events.
```

File: harness_cloud_cost_management/README.md [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+
+### Service Checks
+
+The Ambassador check does not include any service checks.
```
Comment (sarah-witt) [inline harness_cloud_cost_management/README.md:28]:
```suggestion
The Harness Cloud Cost Management  check does not include any service checks.
```

File: harness_cloud_cost_management/manifest.json (hunk 16 above)
Comment (akashbdj) [reply harness_cloud_cost_management/manifest.json:16]: Hi @mgarabed 

Thanks for the suggestion! Could you please explain why do we need to change 'Management' to 'Mgmt'? Is this text shown anywhere in the UI?

**Review by kayayarai — APPROVED** [review]

**Pull request merged**